example/main/src/main/scala/org/finos/vuu/SimulMain.scala

```diff
@@ -56,24 +56,25 @@ object SimulMain extends App with StrictLogging {
   val defaultConfig = ConfigFactory.load()
 
   //look in application.conf for default values
-  val webRoot = defaultConfig.getString("vuu.webroot")
-  val certPath = defaultConfig.getString("vuu.certPath")
-  val keyPath = defaultConfig.getString("vuu.keyPath")
+  private val webRoot = defaultConfig.getString("vuu.webroot")
+  private val certPath = defaultConfig.getString("vuu.certPath")
+  private val keyPath = defaultConfig.getString("vuu.keyPath")
+  private val sslEnabled = defaultConfig.getBoolean("vuu.ssl")
 
   val config = VuuServerConfig(
     VuuHttp2ServerOptions()
       //only specify webroot if we want to load the source locally, we'll load it from the jar
       //otherwise
       .withWebRoot(webRoot)
-      .withSsl(certPath, keyPath)
+      .withSsl(sslEnabled, certPath, keyPath)
```

Comment:
Would it be possible to not call this method at all, or indeed call .withUnencryptedWs()? 
It looks a bit odd calling withSsl, passing in certs, but not using them....

Reply:
Decided to not call this when ssl is disabled. Can always add another method to do `withUnencryptedWs or withoutWss` if/when needed. Let me know what you think?

Reply:
Ok, that is fine to get this over the line. Can we add another ticket to make opting out of Ssl explicit pls? 

The reason I say is if you don't have to choose explicitly one or the other, we may well find lots of poeple deploying unencrypted websockets or http connections without really understanding they've done that. 

It would be good to have a project under the examples/ folder that has unencrypted websockets/http and explains in a readme what our intended use of this would be (i.e. only in containerized environments where the ssl termination occurs in a sidecar setup at the pod level.)

As said, am happy to approve for now, but lets create another ticket to track this. 

Chris

Reply:
Yeah makes sense. Let me do that

Reply:
Done: https://github.com/finos/vuu/issues/1258